src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/caller/CalledCopyRatioSegment.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.caller;
+
+import org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation.CopyRatioSegment;
+import org.broadinstitute.hellbender.utils.Utils;
+
+public class CalledCopyRatioSegment extends CopyRatioSegment {
+    public enum Call {
+        AMPLIFICATION_CALL("+"),
+        DELETION_CALL("-"),
+        NEUTRAL_CALL("0");
+
+        private final String outputString;
+
+        Call(final String outputString) {  this.outputString = outputString; }
+
+        public String getOutputString() {
+            return outputString;
+        }
+    }
+
+    private final Call call;
+
+    public CalledCopyRatioSegment(final CopyRatioSegment segment,
+                                  final Call call) {
```

Comment:
No new line.

Reply:
I prefer putting arguments on new lines.  It makes editing the signatures by hand a bit easier (since you can just delete the entire line), is a bit more readable, and prevents the code from getting too wide (and avoids the related question of when to add line breaks between arguments).